scripts/end_to_end_test/end_to_end_test.yaml

```diff
@@ -239,3 +239,26 @@ steps:
   - sleep: 5
   - run_python_test: { "script": ".smart_contract_tests.malleability_test" }
 ...
+
+---
+test_description: >
+  DISABLED - Test synergetic contract fees
```

Comment:
I guess you are waiting for the new version to be published?

Reply:
yes

Reply:
0.11.0-a1?